tee-worker/core-primitives/types/src/lib.rs

```diff
@@ -47,7 +47,7 @@ pub type CallIndex = [u8; 2];
 
 // pallet teerex
 pub type ConfirmCallFn = (CallIndex, ShardIdentifier, H256, Vec<u8>);
-pub type ShieldFundsFn = (CallIndex, Vec<u8>, Balance, ShardIdentifier);
+pub type ShieldFundsFn = (CallIndex, Vec<u8>, codec::Compact<u128>, ShardIdentifier);
```

Comment:
We should have used `codec::Compact<Balance>` if that's possible - but since we are not going to change shielding_funds - let's just revert the changes